README.md

```diff
@@ -61,6 +61,25 @@ The release process is building all runtimes and then puts them into a release i
 
 The format of [`CHANGELOG.md`](CHANGELOG.md) is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Release guidelines
+
+Here are the recommended steps for both major and minor runtime releases.
+
+|Steps |Description |Major |Minor |
+|------|------------|------|------|
+|0 |Open an [issue](https://github.com/polkadot-fellows/runtimes/issues) for the release in the runtimes repo with the **version number**. | ✔️ | ✔️ |
```

Comment:
I think we should have a step where we fix the features/fixes at least that break APIs in that release issue. We can use one primary notification channel to announce at least those two steps: 1. started preparing a release candidate with issue link, 2. release fixed, here the list of breaking changes. 
@anaelleltd I would like to schedule some call, we can work a bit more on this guideline